packages/lmi/tests/cassettes/TestLiteLLMEmbeddingModel.test_caching.yaml

```diff
@@ -1,215 +0,0 @@
-interactions:
```

Comment:
Can we bring these cassettes back? We need them, what's happening now is CI itself just regens the cassettes

Also, why are embedding cassettes being changed, they're unrelated to `top_logprobs`

Reply:
https://github.com/Future-House/ldp/pull/330